### ISSUE TITLE
paragraph not apeare

### ISSUE THREAD

**Issue by ulfetalhussaini** [PiranhaCMS/piranha.core#928]
where can i edit this?
(in release  v7.1)
![q](https://user-images.githubusercontent.com/38277711/69532497-05c4b300-0f87-11ea-8915-126804856246.jpeg)

**Comment by tidyui** [comment]
I see you're using the brand new `Summernote` module. We currently don't have any built in quick configuration of this editor. The simplest way is to just override the init script with your own `.js` file. All you need to override is:

~~~ js
piranha.editor.addInline = function (id, toolbarId, cb) {
  ...
}
~~~

You can find the standard implementation here:

https://github.com/PiranhaCMS/piranha.core/blob/master/core/Piranha.Manager.Summernote/assets/src/piranha.editor.js#L45

If it as generic error in the setup you are more than welcome to send a PR with your modifications.

Best regards

**Comment by ulfetalhussaini** [comment]
In this last update [https://github.com/PiranhaCMS/piranha.core](url) you solved it and it works, but in release v7.1 there is an issue ..
I want to know how you solved it, to make edit in release version cause i have made my own edits on it. 

![p](https://user-images.githubusercontent.com/38277711/69605943-8ee4f400-1032-11ea-92d2-80be51cd6967.jpeg)

**Comment by tidyui** [comment]
The screenshots from the example project you are showing are using completely different editors. The first screenshot is using the **Summernote** editor, the second one is using the **TinyMCE** editor. They look and behave differently.

1. Make sure you include the project package you want (https://github.com/PiranhaCMS/piranha.core/blob/master/examples/RazorWeb/RazorWeb.csproj#L22)
2. Add the included editor to your Startup (https://github.com/PiranhaCMS/piranha.core/blob/master/examples/RazorWeb/Startup.cs#L31) (https://github.com/PiranhaCMS/piranha.core/blob/master/examples/RazorWeb/Startup.cs#L92)
